### DISCUSSION TITLE
sdrpp: error while loading shared libraries: libvolk.so.1.4

### DISCUSSION THREAD

**Discussion by emfi2r** [AlexandreRouma/SDRPlusPlus#150] [Q&A]
My failed results tring to install and run SDRPlusPlus on Linux Mint 20.1 (On my Dell Latitude D820 laptop):

dad@Dad-Dell-Latitude-D820-Linux-Mint:~/sdrpp$ sudo dpkg -i sdrpp_debian_amd64.deb
Selecting previously unselected package sdrpp.
(Reading database ... 334235 files and directories currently installed.)
Preparing to unpack sdrpp_debian_amd64.deb ...
Unpacking sdrpp (0.2.5) ...
Setting up sdrpp (0.2.5) ...
dad@Dad-Dell-Latitude-D820-Linux-Mint:~/sdrpp$ sdrpp
sdrpp: error while loading shared libraries: libvolk.so.1.4: cannot open shared object file: No such file or directory

I tried this solution from https://www.gitmemory.com/issue/AlexandreRouma/SDRPlusPlus/71/782690588

dad@Dad-Dell-Latitude-D820-Linux-Mint:~/sdrpp$ sudo ln -s /usr/lib/x86_64-linux-gnu/libvolk.so /usr/lib/x86_64-linux-gnu/libvolk.so.1.4
dad@Dad-Dell-Latitude-D820-Linux-Mint:~/sdrpp$ sdrpp
[2021-06-21 12:24:59.208] [info] SDR++ v0.2.5_beta
[2021-06-21 12:24:59.209] [warning] Root directory /home/dad/.config/sdrpp does not exist, creating it
[2021-06-21 12:24:59.209] [info] Loading config
[2021-06-21 12:24:59.209] [warning] Config file '/home/dad/.config/sdrpp/config.json' does not exist, creating it
[2021-06-21 12:24:59.793] [error] Glfw Error 65543: GLX: Failed to create context: GLXBadFBConfig

Any way to fix this?

**Comment by AlexandreRouma** [comment]
this is an old release, you can get a recent deb from the automated build system this way:

1) Go to [this page](https://github.com/AlexandreRouma/SDRPlusPlus/actions?query=event%3Apush+is%3Asuccess+branch%3Amaster+actor%3AAlexandreRouma)
2) Click on the latest build  (aka the one at the top)
3) Scroll down to artifacts and download the one most appropriate for your distro  (Ubuntu focal in your case).
